src/test/resources/repo_permissions.yml

```diff
@@ -9,12 +9,12 @@ repo:
     # john can deploy and delete
     # jane can only deploy
     # any user can download
-    - admin:
-        - "*"
-    - john:
-        - deploy
-        - delete
-    - jane:
-        - deploy
-    - "*":
-        - download
\ No newline at end of file
+    admin:
```

Comment:
@olenagerasimova Im thinking about testing "negative" scenarios which can be possibly defined in yml file to validate that no exceptions are raised. For instance 
```
- john:
    -
    - delete
- jane:
```
WDYT is it worth testing?

Reply:
@Vatavuk hm, seems like we will be testing `com.amihaiemil.eoyaml`, not our permissions settings...

Reply:
@olenagerasimova  There is also a security dimension involved here. If parsing is not right `jane` or `john` might be able to access everything. I believe it is valuable to assert this even though it might seem that we are validating `com.amihaiemil.eoyaml` but actually we are testing different permission inputs to assert that everything works as expected. `RpPermissions` implementation might change in the future and this test would act as an additional safety net.

Reply:
@Vatavuk hm, not sure I agree, but I've added such case. Please note, that empty strings in yaml should be replaced with ~, otherwise yaml is simply invalid.

Reply:
@olenagerasimova ok great